packages/tentales/src/index.d.ts

```diff
@@ -87,6 +94,9 @@ declare module "tentales" {
   export interface Config {
     port: number
     public: boolean
+    auth: {
+      serverSecret: string | Buffer
```

Comment:
When would this be of type `Buffer`?

Reply:
If a certificate is used instead of a string. E.g. 

```
serverSecret: fs.readFileSync('private.key')
```

Reply:
Wet cool, didn’t think about that